### ISSUE TITLE
Hosting Locally-- ActiveSupport::Duration can't be coerced into Float

### ISSUE THREAD

**Issue by lilliankorinek** [publiclab/plots2#1504]
### What happened just before the problem occurred

I followed the directions for installation found here: https://github.com/publiclab/plots2/.

1. An error occurred after running rake db:setup. The following was listed: 
 
````
Lillians-MBP:plots2 lilliankorinek$ rake db:setup
rake aborted!
Gem::LoadError: You have already activated rake 12.0.0, but your Gemfile requires rake 10.5.0. Prepending `bundle exec` to your command may solve this.
/usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:317:in `check_for_activated_spec!'
/usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:32:in `block in setup'
/usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:27:in `map'
/usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/runtime.rb:27:in `setup'
/usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler.rb:101:in `setup'
/usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.1/lib/bundler/setup.rb:10:in `<top (required)>'
/Users/lilliankorinek/plots2/config/boot.rb:6:in `<top (required)>'
/Users/lilliankorinek/plots2/config/application.rb:1:in `<top (required)>'
/Users/lilliankorinek/plots2/Rakefile:5:in `<top (required)>'
/usr/local/lib/ruby/gems/2.4.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
LoadError: cannot load such file -- bundler/setup
/Users/lilliankorinek/plots2/config/boot.rb:6:in `<top (required)>'
/Users/lilliankorinek/plots2/config/application.rb:1:in `<top (required)>'
/Users/lilliankorinek/plots2/Rakefile:5:in `<top (required)>'
/usr/local/lib/ruby/gems/2.4.0/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
(See full trace by running task with --trace)
Lillians-MBP:plots2 lilliankorinek$ 
````
I suspect the issue arises from "LoadError: cannot load such file -- bundler/setup"?

2. After running bundle exec the issue continued. This time I ran into new errors: 

"TypeError: Cannot visit Integer"
"NoMethodError: undefined method `visit_Integer' for #<Arel::Visitors::SQLite:0x007f85c50b3db0>
Did you mean?  visit_String"

**Comment by icarito** [comment]
Hi,
I suspect the reason is that you have a system with a newer Ruby. Currently Plots runs with Ruby 2.1.2. Here are the [prerequisites](https://github.com/publiclab/plots2/wiki/Prerequisites). You may use rvm to install it.
Perhaps we should put a note about it in the main instructions. :confused:

**Comment by ryzokuken** [comment]
@lilliankorinek try prefixing all of your `rake` commands with `bundle exec`. If they still fail, post the new error log and hopefully we'll help you out. This is a common issue which happens with mismatched rake versions. I would strongly advise you to use tools like rvm and hardcode the versions of the tools you use to the ones we recommend, as unmatched versions may also cause problems later on.

**Comment by lilliankorinek** [comment]
@ryzokuken Thank you so much! I'll try it now and report back if the problems continue. Again, thank you so much for your help!

**Comment by jywarren** [comment]
http://rvm.io is pretty easy to install!

**Comment by ryzokuken** [comment]
@lilliankorinek you never let us know what happened :sweat_smile:

**Comment by lilliankorinek** [comment]
@ryzokuken The recommendations you offered helped solve it! Bundle Exec was the answer, but only after I ensured the prerequisites were downloaded. 

Thank you for directing me to the prerequisite page. I was able to read through Jasmine and get a better understanding of what I needed. The website can be hosted locally on my machine! Thank you Thank you

**Comment by ryzokuken** [comment]
@lilliankorinek that's awesome. I am closing this one then, if you face any further issues, please feel free to ask on the chatroom or open another issue.
